tests/cli/test_family.py

```diff
@@ -57,28 +74,28 @@ def test_family_cutoffs_set(run_cli_command, get_pseudo_family, generate_cutoffs
 
     # Set correct stringency
     stringency = 'high'
-    filepath.write_text(json.dumps(cutoffs['high']))
+    filepath.write_text(json.dumps(cutoffs_dict['high']))
     result = run_cli_command(cmd_family_cutoffs_set, [family.label, str(filepath), '-s', stringency])
     assert 'Success: set cutoffs for' in result.output
     assert stringency in family.get_cutoff_stringencies()
     assert sorted(family.get_cutoff_stringencies()) == sorted(['low', 'normal', 'high'])
-    assert family.get_cutoffs(stringency) == cutoffs[stringency]
+    assert family.get_cutoffs(stringency) == cutoffs_dict[stringency]
 
 
 @pytest.mark.usefixtures('clear_db')
 def test_family_cutoffs_set_unit(run_cli_command, get_pseudo_family, generate_cutoffs, tmp_path):
     """Test the `aiida-pseudo family cutoffs set` command with the ``--unit`` flag."""
     family = get_pseudo_family(cls=CutoffsFamily)
     stringency = 'normal'
-    cutoffs = generate_cutoffs(family, stringencies=(stringency,))
+    cutoffs = generate_cutoffs(family)
 
     # Currently, the CLI checks that the unit set matches the one already set on the family, because only a single
     # unit can be used at a time. This limitation will soon be removed though, at which point, this family should have
     # eV as a unit, such that the test of the CLI call to set it to hartree actually tests that the unit is changed.
-    family.set_cutoffs(cutoffs, unit='hartree', default_stringency=stringency)
+    family.set_cutoffs(cutoffs, stringency, 'hartree')
```

Comment:
```suggestion
    family.set_cutoffs(cutoffs, stringency, 'eV')
```
use an explicitly different unit than the one we set later for the actual test. This is now possible because we allow multiple units to be specified. You can therefore also remove the three comments above this, or replace them with an explanation why we set `eV` explicitly (this is just in case in the future for some reason the default unit is changed to `hartree`)

Reply:
Ah, I missed that comment while swashbuckling through the tests somehow. My bad!